### PR TITLE
Ugrade sdk to v0.45.7

### PR DESCRIPTION
**Pull request by alpe**
This is the same as #937 without ibc-go upgrade. All credits go to @faddat  ❤️

### CODE REVIEW

**Comment by faddat** [comment]
nice!  This will help with database performance, thank you!

File: app/test_helpers.go [hunk 1]
```diff
@@ -343,6 +345,7 @@ func SignAndDeliver(
 	chainID string, accNums, accSeqs []uint64, expSimPass, expPass bool, priv ...cryptotypes.PrivKey,
 ) (sdk.GasInfo, *sdk.Result, error) {
 	tx, err := helpers.GenTx(
+		rand.New(rand.NewSource(time.Now().UnixNano())),
```
Comment (pinosu) [inline app/test_helpers.go:348]: 👍

File: benchmarks/app_test.go [hunk 2]
```diff
@@ -159,7 +161,8 @@ func InitializeWasmApp(b testing.TB, db dbm.DB, numAccounts int) AppInfo {
 		Msg:    initBz,
 	}
 	gasWanted := 500000 + 10000*uint64(numAccounts)
-	initTx, err := helpers.GenTx(txGen, []sdk.Msg{&initMsg}, nil, gasWanted, "", []uint64{0}, []uint64{1}, minter)
+	initTx, err := helpers.GenTx(rand.New(rand.NewSource(time.Now().UnixNano())),
```
Comment (pinosu) [inline benchmarks/app_test.go:164]: 👍

**Review by pinosu — APPROVED** [review]
LGTM 💯

**Pull request merged**